WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderResourceVars.java

```diff
@@ -29,6 +31,9 @@
         Resources resources = context.getResources();
 
         int displayWidthPx = DisplayUtils.getDisplayPixelWidth(context);
+
+        canShowTiledGallery = DisplayUtils.pxToDp(context, displayWidthPx) > 640;
```

Comment:
Oops, meant to rehash what was in the code so, I'll edit the PR description. Thanks!